src/analytics.ts

```diff
@@ -48,13 +48,14 @@ export function createAnalyticsInstance(options?: Options) {
                         device_type: growthbookOptions?.attributes.device_type,
                     }),
                     ...(growthbookOptions?.attributes?.url && { url: growthbookOptions?.attributes.url }),
+                    ...(growthbookOptions?.attributes?.user_ip && { url: growthbookOptions?.attributes.user_ip }),
                 }
             growthbookOptions ??= {}
             growthbookOptions.attributes ??= {}
             growthbookOptions.attributes.id ??= _rudderstack.getAnonymousId()
             growthbookOptions.attributes.country ??=
                 Cookies.get('clients_country') ||
-                (Cookies as any).getJSON('website_status')?.clients_country ||
+                Cookies.getJSON('website_status')?.clients_country ||
```

Comment:
Property 'getJSON' does not exist on type 'CookiesStatic<string> & { noConflict?(): CookiesStatic<string>; }'.

We'll need `as any` ?